content/en/docs/installation/installation-windows.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Installation on Windows"
+description: "A description on how to install the GMT on Windows machines"
+lead: ""
+date: 2023-12-04T01:49:15+00:00
+weight: 903
```

Comment:
Shouldn't this be 104 to not conflict with installation-cluster.md?

Reply:
you're right. I changed now the weight of `installation-cluster.md` to 904. I hope that is also ok for you.

Reply:
sure :)